app/models/models.js

```diff
@@ -2315,10 +2346,58 @@ YUI.add('juju-models', function(Y) {
         var relationData = endpoints.map(function(endpoint) {
           return endpoint[0] + ':' + endpoint[1].name;
         });
-        result.envExport.relations.push(relationData);
+        relations.push(relationData);
```

Comment:
there's another card about relations being exported when they're ghosts. Does this check and skip ghost services, machines, relations, and config?